### PR TITLE
base branch with API v.0.7 usage, put migrations into startup script

### PR DESCRIPTION
**Pull request by ivanredbread**
## Description

base branch with API v.0.7 usage, put migrations into startup script

- [x] merge [changes in library](https://github.com/uploadcare/pyuploadcare/pull/222)
- [x] switch `pyuploadcare` dependency to `rev = "main"` before merging into `main` here

## Checklist

- [ ] Tests (if applicable)
- [x] Documentation (if applicable)
- [ ] Changelog stub (or use [conventional commit messages](https://www.conventionalcommits.org/))

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -173,6 +180,58 @@ Conversion result page also includes information about how conversion is going.
 ![Convert video result](./img/video_conversion_status.png)
 
 
+### Addons
+
+Use can execute operations that wrapped as addons.
+There are three implemented addons:
+- background removing 
+- virus scanning
+- labels recognition
```
Comment (rsedykh) [inline README.md:189]:
```suggestion
- object recognition
```

File: README.md [hunk 2]
```diff
@@ -173,6 +180,58 @@ Conversion result page also includes information about how conversion is going.
...
+
+For additional information proceed to [Uploadcare REST API Add-ons](https://uploadcare.com/api-refs/rest-api/v0.7.0/#tag/Add-Ons).
+
+
+#### Labels recognition via AWS
```
Comment (rsedykh) [inline README.md:194]:
```suggestion
#### Object recognition via AWS
```

**Review by rsedykh — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -173,6 +180,58 @@ Conversion result page also includes information about how conversion is going.
 ![Convert video result](./img/video_conversion_status.png)
 
 
+### Addons
+
+Use can execute operations that wrapped as addons.
```
Comment (rsedykh) [inline README.md:185]:
```suggestion
User can execute operations that wrapped as addons.
```

Or you wanted to use "You"?

**Review by rsedykh — CHANGES_REQUESTED** [review]

**Review by rsedykh — APPROVED** [review]
It would be lovely to avoid non en localization in screenshots. E.g. img/file_upload.png for choose file.

**Comment by ivanredbread** [comment]
Feel free to rewrite this file with correct one if your system locale provides an opportunity to make screenshot with preferable langauge!

**Pull request merged**